### PR TITLE
Fix classloader related test failures

### PR DESCRIPTION
**Pull request by yuzhu**

### CODE REVIEW

**Comment by yuzhu** [comment]
jenkins, test this with java 11 please

**Comment by yuzhu** [comment]
jenkins, test this with java 11 please

**Comment by yuzhu** [comment]
jenkins, test this with java 11 please

**Comment by yuzhu** [comment]
@bradyoo this reduces the number of errors under java 11 to 8, PTAL

**Comment by yuzhu** [comment]
jenkins, test this with java 11 please

File: core/common/src/main/java/alluxio/util/ClassLoaderUtils.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+public class ClassLoaderUtils {
+  /**
+   * Add a path to be loaded.
+   * @param path full path
+   */
+  public static void addURL(String path) throws Exception {
+    ClassLoader classLoader = ClassLoader.getSystemClassLoader();
+    try {
+      Method method = classLoader.getClass().getDeclaredMethod("addURL", URL.class);
+      method.setAccessible(true);
```
Comment (bradyoo) [inline core/common/src/main/java/alluxio/util/ClassLoaderUtils.java:34]: This is only used in tests right? if so, I would put this in the test and use WhiteBox.

Comment (bradyoo) [reply core/common/src/main/java/alluxio/util/ClassLoaderUtils.java:34]: Actually, why does WhiteBox.invokeMethod fail?

File: core/common/src/main/java/alluxio/util/ClassLoaderUtils.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      method.invoke(classLoader, new File(path).toURI().toURL());
+    } catch (NoSuchMethodException e) {
+      Method method = classLoader.getClass().getDeclaredMethod(
+          "appendToClassPathForInstrumentation", String.class);
```
Comment (bradyoo) [inline core/common/src/main/java/alluxio/util/ClassLoaderUtils.java:38]: Oh I see.. I might just call WhiteBox.invokeMethod twice

File: core/common/src/main/java/alluxio/util/ClassLoaderUtils.java (hunk 1 above)
Comment (yuzhu) [reply core/common/src/main/java/alluxio/util/ClassLoaderUtils.java:34]: if our goal is to eventually remove the powermock dependency,  why would we use Whitebox? 
I will move this util class to somewhere in test package.

**Comment by yuzhu** [comment]
alluxio-bot, test this please

**Comment by bradyoo** [comment]
LGTM

**Comment by yuzhu** [comment]
alluxio-bot, merge this please

**Pull request merged**